src/components/Executions/ExecutionDetails/useExecutionNodeViewsState.ts

```diff
@@ -26,11 +28,18 @@ export function useExecutionNodeViewsState(
     };
 
     const shouldEnableQuery = (executions: NodeExecution[]) =>
-        every(executions, nodeExecutionIsTerminal) &&
-        executionIsTerminal(execution);
+        !executionIsTerminal(execution) ||
+        some(executions, ne => !nodeExecutionIsTerminal(ne));
```

Comment:
nit: any reason to use the JS native `ARRAY.some`?

Reply:
Only that I wasn't aware it existed :-)